modules/apps/frontend-js/frontend-js-state-web/src/main/resources/META-INF/resources/State.ts

```diff
@@ -502,23 +502,3 @@ const State = {
 function isAtom(value: unknown): value is Atom<any> {
 	return Object.hasOwnProperty.call(value, ATOM);
 }
-
-window.Liferay = window.Liferay || {};
-
-window.Liferay.State = State;
-
```

Comment:
We still need these lines, as they are adding State to the global object for runtime